python_modules/libraries/dagster-aws/dagster_aws/s3/resources.py

```diff
@@ -91,6 +97,7 @@ def example_s3_solid(context):
         region_name=context.resource_config.get("region_name"),
         endpoint_url=context.resource_config.get("endpoint_url"),
         use_unsigned_session=context.resource_config["use_unsigned_session"],
+        profile_name=context.resource_config["profile_name"],
```

Comment:
If we stop defaulting `profile_name`, we'll want to switch this to use `.get()` like we do for the other completely optional configuration.

Reply:
Done!